analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/DatabasePasswordsShouldBeSecure/Values/Web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <connectionStrings>
+    <add name="myConnection1" connectionString="Server=myServerAddress;Database=myDataBase;User Id=myUsername;Password=" /> <!-- Noncompliant {{Use a secure password when connecting to this database.}} -->
+    <add name="myConnection2" connectionString="PASSWORD=" /> <!-- FN -->
```

Comment:
Should we consider also [external connection strings](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings-and-configuration-files#using-external-configuration-files) and [encrypted connection strings](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings-and-configuration-files#encrypting-configuration-file-sections-using-protected-configuration)?

Reply:
for external connection strings I was expecting to have a separate file with connection strings and in the main `web.config` file a source defined: `<connectionStrings configSource="connections.config"/>`.

Reply:
Thanks, sorry, I missed that.
I will not include supporting external config. Adding an FN test case.